### PR TITLE
Temporarily disable UWP CI

### PR DESCRIPTION
**Pull request by antkmsft**
Our UWP CI legs are currently blocked until https://github.com/microsoft/vcpkg/issues/35279 is fixed / https://github.com/microsoft/vcpkg/pull/35116 is merged, so if your PR is blocked, you can merge this PR to unblock. We will revert it once it is fixed in vcpkg.

### CODE REVIEW

**Review by ahsonkhan — APPROVED** [review]
Approving to unblock CI, let's get an issue to track bringing this back.

File: eng/pipelines/templates/stages/platform-matrix.json [hunk 1]
```diff
@@ -107,37 +107,6 @@
...
-          "PublishMapFiles": "true"
-        }
-      },
-      "TargetPlatform": {
-        "UWP_debug": {
-          "CMAKE_SYSTEM_NAME": "WindowsStore",
-          "CMAKE_SYSTEM_VERSION": "10.0",
-          "CmakeArgs": " -DBUILD_TRANSPORT_WINHTTP=ON -DDISABLE_AZURE_CORE_OPENTELEMETRY=ON ",
-          "BuildArgs": "--parallel 8 --config Debug"
-        },
```
[the remark this answers is not in the document]
Comment (benbp) [reply eng/pipelines/templates/stages/platform-matrix.json:126]: @ahsonkhan do you mean in the config, or on an ad-hoc basis? For the latter, you can set a the pipeline variable `jobMatrixFilter` with a regex for the test job title(s) to run.

**Pull request merged**